third_party/docfx/templates/devsite/layout/_master.tmpl

```diff
@@ -5,18 +5,10 @@
 {{!include(logo.svg)}}
 {{!include(search-stopwords.json)}}
 <!DOCTYPE html>
-<!--[if IE]><![endif]-->
 <html devsite>
   {{>partials/head}}
   <body data-spy="scroll" data-target="#affix" data-offset="120">

```

Comment:
I don't think we need the `data-*` attributes.

Reply:
done, left to just <body>